js/components/controlBar.js

```diff
@@ -231,20 +186,12 @@ var ControlBar = React.createClass({
         onClick={this.handleVolumeClick}></a>);
     }
 
-    var volumeHeadPaddingStyle = {};
-    volumeHeadPaddingStyle.left = parseFloat(this.props.controller.state.volumeState.volume) * 100 + "%";
-    var volumeIndicatorStyle = {};
-    volumeIndicatorStyle.width = volumeHeadPaddingStyle.left;
-
-    var volumeSlider = [];
-    volumeSlider.push(
-      <div className="volumeSlider" ref="volumeSlider" onTouchEnd={this.handleVolumeBarTouchEnd} key={i}>
-        <div className="volumeIndicator" style={volumeIndicatorStyle}></div>
-        <div className="playheadPadding" style={volumeHeadPaddingStyle}
-          onTouchStart={this.handleVolumeHeadTouchStart}>
-          <div className="volumeHead"></div>
-        </div>
-      </div>);
+    var volumeSlider = <div className="volumeSlider"><Slider volumeSliderValue={parseFloat(this.props.controller.state.volumeState.volume)}
+                        changeVolumeSlider={this.changeVolumeSlider}
```

Comment:
Let's use generic prop names, like:
  * `volumeSliderValue ` to `value`
  * `changeVolumeSlider ` to `onChange `